lib/views/summary.erb

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+  </head>
+  <body>
+
+    <div class="container">
+      <div class="row">
+        <div class="col" >
+          <% @customer.each  do |key|%>
+          <%= key.to_s %>
+          <%= @customer[:key] %>
+          <% end %>
+        </div>
+        <div class="col">
+          <% @customer.each  do |key|%>
+          <%= key.to_s %>
+          <%= @customer[:key] %>
+          <% end %>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+
+ <table class="table table-striped">
+  <thead>
+    <tr>
+      <th scope="col">ID</th>
+      <th scope="col">Name</th>
+      <th scope="col">Price</th>
+      <th scope="col">Quantity</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @items.each do |item| %>
+    <tr>
+      <td><%= item[:part_id] %></td>
+      <td><%= item[:part_name] %></td>
+      <td>£ <%= item[:part_price].to_f.round(2) %></td>
```

Comment:
Maybe deal with the price elsewhere and return a string instead of doing it here?

Reply:
We're going to need to deal with converting prices with our business logic for calculating totals.
Probably best to hold off on doing this in your code as it'll probably duplicate what we need to do.